content/attacks/posts/2022-10-06-BNB-Chain's-exploit

```diff
@@ -0,0 +1,32 @@
+---
+date: 2022-10-06
+custodians: Binance
+categories: Protocol exploit
+title: "BNB chain’s cross-chain bridge exploited, $100 million stolen"
+---
+
+## Summary
+
+On October 6, 2022, the cross-chain bridge responsible for Binance Coin (BNB) ecosystem operation was hacked. The Attacker illegaly minted 2 million BNB, worth approximately $566 million. The Attacker then utilized various liquidity providers and lending protocols borrowing stablecoins and using previouly issued BNB as collateral. Although Binance did its best and froze most of the funds on BSC, hacker still managed to walk away with ~$137 million USD woth of crypto.
+
+## Attackers
```

Comment:
if you have a link to wallet addresses please add the addresses and the link. if not no worries.

Reply:
Wanted to mention that, this article is the same with mine, called BSC-Token-Hub. But if this one looks better, i am okay with closing mine 👌🏻

Reply:
thank you for pointing this out. Also depends on timing for complete submission.